From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: EtcPot <et2416444244@outlook.com>
Date: Sat, 26 Sep 2026 17:00:00 +0800
Subject: [PATCH 1/3] Rebrand AOSP strings to EtcPot OS

Patch framework-res to rename "Android" user-visible strings to
"EtcPot OS". Applied on top of AOSP 16.x.
---
 .../base/core/res/res/values/strings.xml     | 22 ++++++++++++------
 1 file changed, 16 insertions(+), 6 deletions(-)

diff --git a/frameworks/base/core/res/res/values/strings.xml b/frameworks/base/core/res/res/values/strings.xml
--- a/frameworks/base/core/res/res/values/strings.xml
+++ b/frameworks/base/core/res/res/values/strings.xml
@@ -1,12 +1,22 @@
-<!-- Stock AOSP strings -->
-<string name="android_version">Android 16</string>
-<string name="power_dialog_title">Phone options</string>
-<string name="reboot_to_recovery">Reboot to recovery</string>
+<!-- EtcPot OS rebrand -->
+<string name="android_version">EtcPot OS 1.0</string>
+<string name="power_dialog_title">EtcPot OS</string>
+<string name="reboot_to_recovery">重启到 EtcPot Recovery</string>
+<string name="volume_dialog_title">EtcPot 音量</string>
--
2.34.1
